src/main/java/fi/dy/masa/litematica/scheduler/tasks/TaskCountBlocksPlacement.java

```diff
@@ -49,7 +52,7 @@ protected void countAtPosition(BlockPos pos)
             {
                 this.countsMissing.addTo(stateSchematic, 1);
             }
-            else if (stateClient != stateSchematic)
+            else if ((this.ignoreState && stateClient.getBlock() != stateSchematic.getBlock()) || (this.ignoreState == false && stateClient != stateSchematic))
```

Comment:
```
if (this.ignoreState ? stateClient.getBlock() != stateSchematic.getBlock() : stateClient != stateSchematic)
```
This should be more readable...

Reply:
Oh, yeah, that's indeed a nicer way of doing it. @maruohon I've already made this change, let me know if you want it pushed.

Reply:
Yeah that looks nicer

Reply:
It's done!

Reply:
You should be able to mark conversation as resolved. It would make it collapse on your pull's main page, so it doesn't take up so much space.